api/src/test/java/org/apache/iceberg/expressions/TestExpressionHelpers.java

```diff
@@ -137,16 +137,16 @@ public void testRewriteNot() {
   @Test
   public void testTransformExpressions() {
     Assert.assertEquals("Should produce the correct expression string",
-        "year(ref(name=\"ts\")) == \"2019\"",
+        "year[0](ref(name=\"ts\")) == \"2019\"",
```

Comment:
UTC should continue to produce the old names for compatibility. Otherwise, this will break old readers.